Python/pdf_reorder/pdf_reorder_.py

```diff
@@ -0,0 +1,55 @@
+from pdfrw import PdfReader, PdfWriter
+"""
+The function input_and_parse gathers the inputs and parses and then sorts them in
+the order required by reorder(path,dic) function
+It takes one argument which is page length of pdf 'n'
+"""
+
+
+def input_and_parse(n):
+    print("enter the current page and the page you want it to be on seperate values by a comma ',' \n")
+    """
+    store the input in a list and then convert the input string into
+    using map function to convert the data in lists into int values
+    """
+    lst = list(map(lambda x: [int(x[0]), int(x[1])],
+                   [input().split(',') for _ in range(n)]))
+    # Swapping the position of the lst values to better parse it in dictionary
+    lst = [[x[1], x[0]] for x in lst]
+    lst.sort(key=lambda x: x[0])
+    dic = {curr: new for curr, new in lst}
+    # now I have sorted the dic to the required needs of reorder function
+    return dic
+
+
+"""
+The function reorder takes two arguments path and dic
+path is the path of the source pdf file which is in wrong
+order
+"""

```

Comment:
Incorrect use of Docstrings. Please take a look over on how Docstrings are formatted and used.

Reply:
Added the docstrings inside the function and also with some subheadings like parameters and return don't whether they are right or not.